### PR TITLE
Upgrade jsonld serialization library

### LINKED ISSUE #3327: Consider replacing @rdfjs/serializer-jsonld-ext with jsonld.js
Currently, we use @rdfjs/serializer-jsonld-ext for serializing JSON. However, this introduces a security vulnerability: https://github.com/digitalbazaar/jsonld.js/issues/457

I believe that @rdfjs/serializer-jsonld-ext uses an old (and vulnerable) version of jsonld.js. For this ticket, attempt to replace @rdfjs/serializer-jsonld-ext with the most recent version of jsonld.js.

### PR DESCRIPTION
**Pull request by mjgiarlo**
Fixes #3327

### CODE REVIEW

**Comment by mjgiarlo** [comment]
CI results:

```
<--- Last few GCs --->

[172:0x4da5970]   423928 ms: Scavenge (reduce) 2005.6 (2073.3) -> 2004.9 (2073.6) MB, 6.0 / 0.0 ms  (average mu = 0.330, current mu = 0.335) allocation failure 
[172:0x4da5970]   423999 ms: Scavenge (reduce) 2007.1 (2075.2) -> 2006.7 (2075.2) MB, 7.2 / 0.0 ms  (average mu = 0.330, current mu = 0.335) allocation failure 
[172:0x4da5970]   424082 ms: Scavenge (reduce) 2009.3 (2076.9) -> 2007.3 (2075.5) MB, 7.7 / 0.0 ms  (average mu = 0.330, current mu = 0.335) allocation failure 


<--- JS stacktrace --->

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
 1: 0xb02d90 node::Abort() [node]
 2: 0xa18123 node::FatalError(char const*, char const*) [node]
 3: 0xceb85e v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [node]
 4: 0xcebbd7 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [node]
 5: 0xeb55b5  [node]
 6: 0xeb60a4  [node]
 7: 0xec4517 v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [node]
 8: 0xec78cc v8::internal::Heap::AllocateRawWithRetryOrFailSlowPath(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [node]
 9: 0xe8eace v8::internal::Factory::NewMap(v8::internal::InstanceType, int, v8::internal::ElementsKind, int) [node]
10: 0x10ba2db v8::internal::Map::RawCopy(v8::internal::Isolate*, v8::internal::Handle<v8::internal::Map>, int, int) [node]
11: 0x10babbb v8::internal::Map::CopyDropDescriptors(v8::internal::Isolate*, v8::internal::Handle<v8::internal::Map>) [node]
12: 0x10bb3a8 v8::internal::Map::ShareDescriptor(v8::internal::Isolate*, v8::internal::Handle<v8::internal::Map>, v8::internal::Handle<v8::internal::DescriptorArray>, v8::internal::Descriptor*) [node]
13: 0x10bbd43 v8::internal::Map::CopyAddDescriptor(v8::internal::Isolate*, v8::internal::Handle<v8::internal::Map>, v8::internal::Descriptor*, v8::internal::TransitionFlag) [node]
14: 0x10bbe57 v8::internal::Map::CopyWithField(v8::internal::Isolate*, v8::internal::Handle<v8::internal::Map>, v8::internal::Handle<v8::internal::Name>, v8::internal::Handle<v8::internal::FieldType>, v8::internal::PropertyAttributes, v8::internal::PropertyConstness, v8::internal::Representation, v8::internal::TransitionFlag) [node]
15: 0x10bde13 v8::internal::Map::TransitionToDataProperty(v8::internal::Isolate*, v8::internal::Handle<v8::internal::Map>, v8::internal::Handle<v8::internal::Name>, v8::internal::Handle<v8::internal::Object>, v8::internal::PropertyAttributes, v8::internal::PropertyConstness, v8::internal::StoreOrigin) [node]
16: 0x10acf5f v8::internal::LookupIterator::PrepareTransitionToDataProperty(v8::internal::Handle<v8::internal::JSReceiver>, v8::internal::Handle<v8::internal::Object>, v8::internal::PropertyAttributes, v8::internal::StoreOrigin) [node]
17: 0x10cfdac v8::internal::Object::AddDataProperty(v8::internal::LookupIterator*, v8::internal::Handle<v8::internal::Object>, v8::internal::PropertyAttributes, v8::Maybe<v8::internal::ShouldThrow>, v8::internal::StoreOrigin) [node]
18: 0x10878d7 v8::internal::JSObject::SetOwnPropertyIgnoreAttributes(v8::internal::Handle<v8::internal::JSObject>, v8::internal::Handle<v8::internal::Name>, v8::internal::Handle<v8::internal::Object>, v8::internal::PropertyAttributes) [node]
19: 0x11f9614  [node]
20: 0x11fc1ee v8::internal::Runtime_CreateObjectLiteral(int, unsigned long*, v8::internal::Isolate*) [node]
21: 0x15cdf39  [node]
Aborted (core dumped)
```

What I thought looked like a straightforward change turns out not to be... 🤔

File: package.json [hunk 1]
```diff
@@ -97,6 +97,7 @@
     "react-textarea-autosize": "^8.3.3",
     "redux": "^4.1.1",
     "redux-thunk": "^2.3.0",
+    "request": "^2.88.2",
```
Comment (mjgiarlo) [inline package.json:100]: Bumping `@rdfjs/serializer-jsonld-ext` made the `request` library fall out as a dependency. Re-added explicitly. Yes, `request` is deprecated. I naïvely tried to swap out `request` for `got`, but apparently this requires more than just a naïve effort so 🤷🏻. Not sure how much of our time this deserves.

**Comment by justinlittman** [comment]
Because nothing is easy ...

**Comment by justinlittman** [comment]
Where do we use request? (Not seeing it with a grep.)

**Comment by justinlittman** [comment]
Ah, in the server. Maybe replace with https://www.npmjs.com/package/express-http-proxy?

**Comment by mjgiarlo** [comment]
@justinlittman 💬 
> Ah, in the server. Maybe replace with [npmjs.com/package/express-http-proxy](https://www.npmjs.com/package/express-http-proxy)?

A past version of this very small PR replaced `request` with `got` and then I saw the weird GC errors in CI... assumed it was that change, rolled it back, and it turns out to be unrelated. I can replace with `got` again, but i'm inclined to wait until we figure out WTF is wrong with the build when bumping the jsonld library dep.

**Pull request closed without merging**